src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateUserSpecificRegistrySettings(CommandSettings command, string
 
         private void UpdateScreenSaverSettings(CommandSettings command, string securityId)
         {
-            if(!command.GetDisableScreenSaver())
+            if(command.GetKeepScreenSaver())
```

Comment:
Whether the screen saver is enabled/disabled and whether the screen saver is on/off are two different things. I'm trying to figure out whether UI testing can be done when the screen saver is on.